pages/settings/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { GetServerSideProps } from "next";
+
+export default function page() {
+  return null;
+}
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  return {
+    redirect: {
+      destination: "/settings/api-keys",
+      permanent: false,
+    },
+  };
+};
```

Comment:
can we do this in our `middleware.ts` or redirects configuration in next config since we're already hitting that for every request, would like to avoid adding an SSR on every click of /settings